bindings/python/htcondor2/_negotiator.py

```diff
@@ -1 +1,180 @@
+from typing import List
 
+from ._common_imports import (
+    classad,
+    Collector,
+    DaemonType,
+)
+
+from .htcondor2_impl import (
+    _negotiator_command,
+    _negotiator_command_return,
+    _negotiator_command_user,
+    _negotiator_command_user_return,
+    _negotiator_command_user_value,
+)
+
+
+class Negotiator():
+
+    def __init__(self, location : classad.ClassAd = None):
+        if location is None:
+            c = Collector()
+            location = c.locate(DaemonType.Negotiator)
+
+        if not isinstance(location, classad.ClassAd):
+            raise TypeError("location must be a ClassAd")
+
+        self._addr = location['MyAddress']
+        # We never actually use this for anything.
+        # self._version = location['CondorVersion']
+
+
+    def deleteUser(self, user : str) -> None:
+        if not isinstance(user, str):
+            raise TypeError("user must be a string")
+        if '@' not in user:
+            # This was HTCondorValueError in version 1.
+            raise ValueError("You must specify the submitter (user@uid.domain)")
+        _negotiator_command_user(self._addr, self._DELETE_USER, user)
+
+
+    def getPriorities(self, rollup : bool) -> List[classad.ClassAd]:
```

Comment:
I know we don't set a default in the current Python bindings, but is there a good reason why not? The CLI tools require setting `-grouprollup`, so I lean towards `rollup = False`.
```suggestion
    def getPriorities(self, rollup : bool = False) -> List[classad.ClassAd]:
```

Reply:
No reason that I'm aware of; I just didn't think about it at all.